### DISCUSSION TITLE
JFreeSVG 5.0.2 released

### DISCUSSION THREAD

**Discussion by jfree** [jfree/jfreesvg#48] [Announcements]
I've just released JFreeSVG 5.0.2, this adds support for the `TextAttribute.TRACKING` attribute.

**Comment by fixxxer77** [comment]
Hi,
I work on a company that so far has used the free version. But now we would like to license it "JFreeSVG Application Runtime Licence" through https://object-refinery.myshopify.com/products/jfreesvg-application-licence

But it seems not to work, your cart says "This store can’t accept payments right now."

How could we proceed the payment?

**Reply by jfree** [reply]
It is now possible to obtain a _JFreeSVG Application Runtime Licence_ via the GitHub sponsor program. The sponsorship button has been enabled on the project page.

**Reply by fixxxer77** [reply]
Ok, so if we go for a runtime license (alternative to GPLv3) for your choice of JFreeSVG, JFreePDF or Orson Charts we need to understand how does the alternative to GPLv3 look like, do you have more information on the option to use JFreeSVG instead of GPLv3?

**Reply by jfree** [reply]
You can review the runtime license here: https://www.jfree.org/jfreesvg/JFreeSVG%20Software%20Licence%202022.pdf
